blocks/technology-table/technology-table.js

```diff
@@ -0,0 +1,70 @@
+function buildTechnologyTable(block) {
+  const firstDivElements = block.querySelectorAll('.technology-table > div');
+  firstDivElements.forEach((element, i) => {
+    const secondLevelDiv = element.querySelectorAll('div > div');
+    secondLevelDiv.forEach((subelement, j) => {
+      if (i === 0) {
```

Comment:
duplicated if - can be moved to https://github.com/hlxsites/elco-aerotop-sg/pull/48/files#diff-1543d3b3cb1020e9d0f09b32b0ec72566b3c2bf9db7ed76fdc899962e05820ecR15